### DISCUSSION TITLE
detection/core/

### DISCUSSION THREAD

**Discussion by giscus** [roboflow/supervision#1292] [Docs]
# detection/core/

A set of easy-to-use utilities that will come in handy in any computer vision project.

https://supervision.roboflow.com/detection/core/

**Comment by aberg2323** [comment]
Is there an example of how to use get_anchors_coordinates(anchor)?

**Reply by LinasKo** [reply]
It will be something like this:

```python
detections = sv.Detections.from_inference(result)
center_points = detections. get_anchors_coordinates(sv.Position.CENTER)
```

The center points will have a shape `(n, 2)` - a set of `x` and `y` for every detected object in `detections`

**Reply by aberg2323** [reply]
You are seriously a life saver! Thank you!!

**Reply by aberg2323** [reply]
I was able to output the center points, however, I wish to save the center point that corresponds to each detection to the csv file that outputs the tracked detections. Right now, my code appends the csv file with all of the detections in the frame in which the object was detected, not the center point that specifically corresponds to that object. Do you have any suggestions on how I should fix this?

**Reply by LinasKo** [reply]
If you store the center point in the data dict of the `Detections`, the `CSVSink` should save it.

For example, in [this Cookbook](https://supervision.roboflow.com/develop/notebooks/serialise-detections-to-csv/#process-video-and-save-detections-to-csv-file) we are setting the "class_name" before saving the detections in a CSV.

While it's part of InferencePipeline callback and assumes you only have person detections at that point, the line we use is:

```
detections.data["class_name"] = np.array(["person" for _ in range(len(detections))])
```

Hope that helps as a starting point.
